src/components/Cards/Card1.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const Card1 = (props) => {
+  return (
+    <div className="polite lg:ml-5 ">
+      <div className="mapscards bg-slate-50  transition-transform ease-in-out duration-300 hover:scale-90 rounded-3xl h-64 lg:h-72 w-[340px] hover:drop-shadow-xl  hover:shadow-blue-900/70  px-12 md:px-10 md:h-72  lg:px-12 ">
```

Comment:
Mapscards don't seem to be used anywhere, always remove unused classes. Also, minimize the use of custom tailwind dimensions.